pxr/imaging/hdSt/materialXFilter.cpp

```diff
@@ -412,14 +407,14 @@ _AddFallbackDomeLightTextureNode(
             HdStPackageFallbackDomeLightTexture(), 
             HdStPackageFallbackDomeLightTexture()));
     const SdfPath domeTexturePath = 
-        hdTerminalNodePath.ReplaceName(_tokens->domeLightFallback);
-    hdNetwork->nodes.insert({domeTexturePath, hdDomeTextureNode});
+        terminalNodePath.ReplaceName(_tokens->domeLightFallback);
```

Comment:
This and a few other related pieces of data are now encapsulated as members of `HdSt_MaterialFilterTask`.